code.json

```diff
@@ -0,0 +1,47 @@
+[
+    {
+      "name": "plio",
+      "organization": "U.S. Geological Survey",
+      "description": "A planetary surface data input/output library written in Python.",
+      "version": "1.5.0",
+      "status": "Release",
+
+      "permissions": {
+        "usageType": "openSource",
+        "licenses": [
+          {
+            "name": "Public Domain, CC0-1.0",
+            "URL": "https://github.com/USGS-Astrogeology/plio/blob/master/LICENSE.md"
+          }
+        ]
+      },
+  
+      "homepageURL": "https://plio.readthedocs.io/en/latest/?badge=latest",
+      "downloadURL": "https://github.com/USGS-Astrogeology/plio/archive/refs/tags/v1.5.0.zip",
+      "disclaimerURL": "https://github.com/USGS-Astrogeology/plio/blob/master/DISCLAMER.md",
+      "repositoryURL": "https://github.com/USGS-Astrogeology/plio/",
```

Comment:
The official URL needs to be a code.usgs.gov repo to comply with FSP. Hopefully, in the future, we will be able to have Git be the repository of record, but in the short term we need to have a code.usgs.gov repo. In the future requires a change to the FSP policy.

Reply:
So, if I am understanding this correctly, I want to: 1. create a mirror on code.usgs.gov and 2. Update the repo url to point to that (or do all the URLS need to point to code.usgs?)